<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btnRegister">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>400</y>
     <width>191</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>REGISTER</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnLoginnow">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>520</y>
     <width>191</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>LOGIN NOW</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>490</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>GO TO</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="enterName">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>80</y>
     <width>441</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="enterId">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>160</y>
     <width>441</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="enterPw">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>240</y>
     <width>441</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="enterGoal">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>320</y>
     <width>441</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
